plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -20,23 +20,20 @@
 
 module Language.UntypedPlutusCore.Evaluation.Machine.Cek
     ( CekValue(..)
+    , CekUserError(..)
     , CekEvaluationException
+    , CekExBudgetState
+    , CekExTally
     , EvaluationResult(..)
     , ErrorWithCause(..)
     , EvaluationError(..)
-    , CekUserError(..)
-    , ExBudget(..)
     , ExBudgetCategory(..)
-    , ExBudgetMode(..)
-    , ExRestrictingBudget(..)
-    , ExTally(..)
-    , CekExBudgetState
-    , CekExTally
```

Comment:
These were confusingly being exported from both the typed and untyped CEK machines, so I removed them and imported the relevant modules directly in `Main.hs`.